domain/src/DomainLoaderInterface.php

```diff
@@ -30,7 +29,7 @@ public function load($id, $reset = FALSE);
   /**
    * Gets the default domain object.
    *
-   * @return Drupal\domain\DomainInterface | NULL
+   * @return \Drupal\domain\DomainInterface | NULL
```

Comment:
Talking to Crell, we probably shouldn't return NULL. Is that coming from our code or from core?